xml/docu_styleguide.smartdocs.xml

```diff
@@ -0,0 +1,353 @@
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       xmlns:its="http://www.w3.org/2005/11/its" version="5.0" xml:id="sec-smartdocs">
+
+ <title>Working with Smart Docs</title>
+ <para>
+  Smart Docs represent the modular approach to documentation whose goal is producing sets of
+  adaptable, solution-based, easy-to-find and reusable documents.
+ </para>
+ <para>
+  Smart Docs consist of smaller information units with the following characteristics:
+ </para>
+<itemizedlist>
+  <listitem>
+   <formalpara>
+    <title>Self-contained</title>
+    <para>
+     Each information unit stands for itself, including requirements, context, instructions, troubleshooting and FAQs.
+    </para>
+   </formalpara>
+  </listitem>
+  <listitem>
+   <formalpara>
+    <title>Cover one distinct topic</title>
+    <para>
+     Rather than covering a complete technology or a set of problems, each information unit
+     serves one distinct purpose. Five different types of information units are provided:
+    </para>
+    </formalpara>
+    <itemizedlist>
+    <listitem>
+     <para>
+      The assembly file builds the actual article from the components listed below.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Task topics instruct the user how to perform certain tasks.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Concept topics explain underlying concepts to the users.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Reference topics inform the user about basic facts (settings, options, parameters, etc.).
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Glue topics help navigate complex topics and provide links to related information
+      (combine texts or structures that do not fit into any other category). Typical
+      glue topics include <quote>For more information</quote> and <quote>What's next</quote> sections.
+      Use glue topics to also add an additional layer of navigation to your article.
+     </para>
+     </listitem>
+   </itemizedlist>
+   </listitem>
+   <listitem>
+   <formalpara>
+    <title>Reused in different contexts</title>
+    <para>
+     Task, concept and reference information units can be reused in different context. Content units
+     (articles) are built by bundling these into dedicated assembly files.
+    </para>
+   </formalpara>
+  </listitem>
+ </itemizedlist>
+  <para>To create a Smart Docs project, use the templates for the assembly and its topical components
+  that are provided in the Doc Modular repository at <link xlink:href="https://github.com/SUSE/doc-modular/tree/main/templates"/>.</para>
+
+<sect2 xml:id="sec-taglist-smartdocs">
+  <title>Using ITS tags with assemblies</title>
+  <para>Assembly files use ITS tags and attributes which flag whether the content of
```

Comment:
Maybe mention what ITS means? I would suggest:

```suggestion
  <para>Assembly files use ITS (International Translation Set) tags and attributes which flag whether the content of
```

Reply:
We already have it explained twice in the previous chapters :-)